### PR TITLE
Sync to nodejs/master (2016-05-20)

### PR DESCRIPTION
**Pull request by kunalspathak**
##### Checklist

- [x] tests and code linting passes
- [x] a test and/or benchmark is included
- [x] the commit message follows commit guidelines

`linebreak-style` rule of Jslint is disabled. See nodejs/node#6912 for more details.
##### Affected core subsystem(s)

test, tools
##### Description of change

Sync to nodejs/master as of (2016-05-20) at 62376d99bd012887dd64b876f20e317bcd6bf6ee

### CODE REVIEW

**Comment by kunalspathak** [comment]
//cc : @nodejs/node-chakracore

File: test/sequential/test-debugger-debug-brk.js [hunk 1]
```diff
@@ -5,6 +5,12 @@ var spawn = require('child_process').spawn;
 
 var script = common.fixturesDir + '/empty.js';
 
+if (common.isChakraEngine) {
+  console.log('1..0 # Skipped: This test is disabled for chakra engine ' +
+  'because debugger support is not implemented yet.');
```
Comment (thefourtheye) [inline test/sequential/test-debugger-debug-brk.js:10]: This should be aligned with the string from the previous line, right? Does the linter allow this?

**Comment by jianchun** [comment]
> linebreaks to be unix style i.e. `LR` by default.`

nit: commit message typo, `LR` should be `LF`

**Comment by jianchun** [comment]
LGTM

File: test/sequential/test-debugger-debug-brk.js (hunk 1 above)
Comment (kunalspathak) [inline test/sequential/test-debugger-debug-brk.js:10]: yes, jslint didn't complain about this.

**Pull request merged**